.github/workflows/build_and_test.yml

```diff
@@ -31,27 +31,32 @@ jobs:
       - name: Run cargo fmt check
         run: cargo fmt --all -- --check
 
-  #  cross:
-  #    runs-on: ubuntu-latest
-  #    steps:
-  #      - name: Clean up files
-  #        run: |
-  #          sudo rm -rf /usr/share/dotnet
-  #          sudo rm -rf /opt/ghc
-  #          sudo rm -rf "/usr/local/share/boost"
-  #          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
-  #
-  #      - name: Add aarch64 target with rustup
-  #        run: rustup target add aarch64-unknown-linux-gnu
-  #
-  #      - name: Install cross tool
-  #        run: cargo install cross
-  #
-  #      - name: Checkout
-  #        uses: actions/checkout@v4
-  #
-  #      - name: Build and test for cross
-  #        run: make cross-ci
+  cross:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clean up files
+        run: |
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /opt/ghc
+          sudo rm -rf "/usr/local/share/boost"
+          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+
+#      - name: Install Linker
+#        run: |
```

Comment:
We can leave the install of the other target, as we will need to build for both….